packages/framework-info/tests/frameworks.test.ts

```diff
@@ -1,4 +1,4 @@
-import { fileURLToPath } from 'url'
+import { fileURLToPath } from 'node:url'
```

Comment:
We should leave it as is, because we are running the tests in CI with node 14.16 which does not support the `node:` prefix.

Reply:
Done; didn't realize the prefix was Node 16+.